src/components/tests/List.spec.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import List from "../List";
+import mockedData from '../../youtube-videos-mock.json';
+
+describe("items list", () => {
+  it("There is an items list displayed", () => {
+    const { items } = mockedData;     
+    render(<List items={items} />);
+    expect(screen.queryByTestId("items-list").tagName).toBe('DIV');
+    expect(screen.queryAllByTestId("items-list").length).toBe(1);
```

Comment:
Doing the following you assert the existence of exactly one element and the content of it:
```suggestion
    expect(screen.queryByTestId("items-list").innerHtml).toBe("any content");
```